### PR TITLE
fix: Translations falling back to english after refreshing the page

### LINKED ISSUE #31802: ignoring the language settings
### Description:
Rocketchat ignores the language settings. By default, I have Russian installed, but when I start the chat, it loads in English. This is fixed if you reboot it ignoring the cache. But the next time you run it, the problem will repeat. This is true for both the web version and the desktop application.

### Steps to reproduce:

1. Start the chat with the Russian language settings.

### Expected behavior:

The chat will start considering the language settings.
<a href="https://ibb.co.com/L9n3X82"></a>
### Actual behavior:

The chat ignores the language settings in the profiles and/or general server settings.
<a href="https://ibb.co.com/vjts8Fq"></a>
### Server Setup Information:

- Version of Rocket.Chat Server:  6.6.1
- Operating System:  synology
- Deployment Method: docker
- Number of Running Instances: 1
- DB Replicaset Oplog: yes
- NodeJS Version: 14.21.3
- MongoDB Version: 6.0.13

### Client Setup Information

- Desktop App or Browser Version: Edge v121.0.2277.128, chrome v121.0.6167.185, desktop app v3.9.14. 
- Operating System: Win 11, Win 10, Win 7

### LINKED ISSUE #32268: Refreshing the page is changing the language back and not respecting the User Preference
### Description:
When a user changes their language preference in the "User Preference" settings and then refreshes the page, the language reverts back to English.

### Steps to reproduce:

1. Go to user preferences
    `/account/preferences`
2. Change to a language different from English
e.g: Belarusian
3. Refresh the page

### Expected behavior:
The User Preference should be respected and the language should be kept as the one selected

### Actual behavior:
It changes the Language on the screen to English (probably because of the browser language) 

Before refreshing:
image

After refreshing:
image

### Server Setup Information:
6.6.3

### Client Setup Information

Desktop App or Browser Version: Browser
Operating System: Windows

### PR DESCRIPTION
**Pull request by gabriellsh**
## Proposed changes (including videos or screenshots)

i18next instance was being initialized twice, making it so that the resources object would be overwritten with only the fallback language which is loaded by default.
## Issue(s)

Closes #32268
Closes #31802 

[CORE-295](https://rocketchat.atlassian.net/browse/CORE-295)
## Further comments

[CORE-295]: https://rocketchat.atlassian.net/browse/CORE-295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tiagoevanp — APPROVED** [review]

File: apps/meteor/tests/e2e/translations.spec.ts [hunk 1]
```diff
@@ -29,8 +29,24 @@ test.describe('Translations', () => {
         expect((await setUserPreferences(api, { language: '' })).status()).toBe(200);
     });
 
-    // Browser language not working on CI for some reason. This test passes locally.
-    test.describe.skip('Browser', async () => {
+    test('expect to keep chosen language after refresh', async ({ page, api }) => {
+        await page.goto('/home');
+        await page.waitForTimeout(5000);
```
Comment (MarcosSpessatto) [inline apps/meteor/tests/e2e/translations.spec.ts:34]: Do we need this fixed timeout? 🤔 Wouldn't it be possible to wait for an element instead?

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]

File: apps/meteor/client/providers/TranslationProvider.tsx [hunk 2]
```diff
@@ -26,7 +25,7 @@ import {
 import { AppClientOrchestratorInstance } from '../../ee/client/apps/orchestrator';
 import { isRTLScriptLanguage } from '../lib/utils/isRTLScriptLanguage';
 
-i18n.use(I18NextHttpBackend).use(initReactI18next).use(sprintf);
+i18n.use(I18NextHttpBackend).use(initReactI18next);
```
Comment (MarcosSpessatto) [inline apps/meteor/client/providers/TranslationProvider.tsx:28]: is it expected to remove the `sprintf` middleware?

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]

File: apps/meteor/client/providers/TranslationProvider.tsx (hunk 2 above)
Comment (ggazzo) [reply apps/meteor/client/providers/TranslationProvider.tsx:28]: `apps/meteor/app/utils/lib/i18n.ts` already use `sprintf`

**Review by MarcosSpessatto — APPROVED** [review]

**Review by dougfabris — APPROVED** [review]

**Review by ggazzo — APPROVED** [review]

**Pull request merged**